src/shared/components/Auth/ForgotPassword.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import styled from "styled-components";
+import { ForgotPassword } from "aws-amplify-react";
+import { IAuthPieceProps } from "aws-amplify-react/lib-esm/Auth/AuthPiece";
+import { AmplifyTheme } from "shared/interfaces/amplify.interface";
+import {
+  Input,
+  Form,
+  Button,
+  FormField,
+  CustomLink,
+  Container,
+} from "shared/components/Auth/common";
+import { capitalizeFirstLetter } from "shared/utils";
+
+const CustomForgotPasswordContainer = styled(Container)`
+  justify-content: center;
+  align-items: center;
+`;
+
+class CustomForgotPassword extends ForgotPassword {
+  constructor(props: IAuthPieceProps) {
+    super(props);
+    this._validAuthStates = ["forgotPassword"];
+  }
+
+  renderRequestCode(theme: AmplifyTheme) {
+    return (
+      <FormField>
+        <label htmlFor={this.props.usernameAttributes}>
+          {capitalizeFirstLetter(this.props.usernameAttributes || "")} *
```

Comment:
https://www.tutorialspoint.com/css/css_text-transform.htm

I think this may be better.